scripts/generate.ts

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env -S deno run -A
+import $ from "https://raw.githubusercontent.com/dsherret/ax/b6dbab0a4c3028538a38fa948df3025605098f19/mod.ts";
+
+if (!Deno.args.some(a => a === "--quick")) {
+  const swcVersions = await getSwcVersions();
+  $.echo(`Setting up crates. Note: Provide --quick to just code generate.`);
+  $.echo(`Setting up swc_ecma_ast ${swcVersions.swcEcmaAst}...`);
+  await $.fs.emptyDir("swc_ecma_ast");
+  await $`cargo clone swc_ecma_ast@${swcVersions.swcEcmaAst}`;
+  await $`cd swc_ecma_ast ; cargo +nightly rustdoc -- --output-format json -Z unstable-options`;
```

Comment:
Haven’t added boolean list support to the shell yet, so a sequential list will do for now.